ModelBundle/Repository/RoleRepository.php

```diff
@@ -35,4 +35,44 @@ public function findOneByFromStatusAndToStatus(StatusInterface $fromStatus, Stat
 
         return $this->singleHydrateAggregateQuery($qa);
     }
+
+    /**
+     * Find the roles that don't contain fromStatus and toStatus
+     *
+     * @return array
+     */
+    public function findAccessRole()
+    {
+        $qa = $this->createAggregationQuery();
+        $qa->match(
```

Comment:
By creating a role without enter a value for fromStatus and toStatus, this role will be present in the Group form. Does't it answer to your question?